docs/en/observability/whats-new.asciidoc

```diff
@@ -1,21 +1,246 @@
+https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
+https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
+https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
 
-// Other versions:
-// {observability-guide-all}/7.10/whats-new[7.10] |
-// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
-// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
-// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
-// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
-// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
-// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+[discrete]
+== Give {minor-version} a try
+
+Try {minor-version} now by deploying {es} and {kib} on
+{ess-product}[Elastic Cloud] or
+by https://www.elastic.co/start[downloading them].
 
 // tag::whats-new[]
 
-// What's new content goes in here.
+[discrete]
+== APM PHP agent (GA)
+
+In {minor-version}, we're excited to announce that the Elastic APM PHP agent is now generally available!
+Thanks to all of you that provided us with valuable feedback on the agent's preview and beta versions.
+The PHP agent adds automatic instrumentation for Web frameworks like Laravel, data access technologies
+like PHP Data Objects (PDO), client-side networking technologies like curl extension and the Guzzle library.
+
+In addition to this out of the box instrumentation, the PHP Agent also provides a comprehensive Public API
+that allows you to custom instrument your PHP applications. If you haven’t had the chance to check out the new agent,
+test drive the agent and let us know what you would like to see added to it.
+
+[discrete]
+== Automatic ECS compliant logs with APM
+
+During the last few releases, we have focused on improving the experience of visualizing your logs along with your
+trace data. This release makes that experience even more seamless. APM agents now do the heavy lifting of creating
+the ECS compliant logs for you.
+
+This means all you have to do is deploy an APM agent to ship your logs to {es} so you can visualize them in {kib}.
+Check out our documentation for a complete list of logging frameworks supported.
+
+[discrete]
+== Prometheus client for custom metrics in APM agents (Python only)
+
+[discrete]
+== Native support for OpenTelemetry (Experimental)
+
+OpenTelemetry is emerging as the standardized collection layer for observability data and is designed to enable
+organizations to instrument their apps in an open and vendor-neutral manner. Our support for OpenTelemetry is not new.
+In July 2020, we released the OpenTelemetry Collector exporter for Elastic that acts as a translation layer between OpenTelemetry agents and APM Server.
+
+[role="screenshot"]
```

Comment:
`[role="screenshot"]` adds a box shadow to images. It makes sense for screenshots, but for diagrams like this, I think it's better to let them blend into the page. Example:

<img width="759" alt="Screen Shot 2021-03-22 at 8 44 55 AM" src="https://user-images.githubusercontent.com/5618806/112017510-eb530900-8aea-11eb-81cd-f8becc3b8c38.png">

---

<img width="762" alt="Screen Shot 2021-03-22 at 8 45 00 AM" src="https://user-images.githubusercontent.com/5618806/112017502-e9894580-8aea-11eb-8444-c14d018f4fc7.png">


```suggestion
```

Reply:
Good point. Thanks!